python/src/trezorlib/debuglink.py

```diff
@@ -221,8 +221,12 @@ def button_request(self, code):
                 self.input_flow = self.INPUT_FLOW_DONE
 
     def get_pin(self, code=None):
```

Comment:
This needs some documentation and a more descriptive name, maybe `get_next_pin()`.

Reply:
this is a UI callback, see https://github.com/trezor/trezor-firmware/blob/master/python/src/trezorlib/ui.py#L61-L130 and https://github.com/trezor/trezor-firmware/blob/master/python/src/trezorlib/client.py#L165-L184